lib/suspenders/app_builder.rb

```diff
@@ -375,6 +375,21 @@ def copy_dotfiles
       directory("dotfiles", ".")
     end
 
+    def download_style_guides
+      download_file(
+        ".jscsrc",
+        "https://raw.githubusercontent.com/thoughtbot/guides/master/style/javascript/.jscsrc",
```

Comment:
@teoljungberg I had the same idea and then decided that I didn't want to box us in to using code from Github. Looking at it again, I think you've sold me.